scripts/eval/yamls/hf_lora_eval.yml

```diff
@@ -5,7 +5,7 @@ precision: amp_fp16
 # If you are using one model, put it here:
 model_name_or_path: EleutherAI/gpt-neo-125m
 # If you are using a seperated lora weight, put it here:
-lora_id_or_path: nathan0/lora-gpt-neo-125m-alpaca
+lora_id_or_path: edbeeching/gpt-neo-125M-imdb-lora
```

Comment:
maybe leave a note that this path is just an example that works with gpt-neo-125M, but the LoRA would be model specific